pkg/domain/infra/abi/play.go

```diff
@@ -98,6 +98,13 @@ func (ic *ContainerEngine) createServiceContainer(ctx context.Context, name stri
 	}
 	s.Name = name
 
+	if len(image) == 0 {
```

Comment:
if the image can be empty here and we use a rootfs we must pass NewSpecGenerator(...,true) to the specgen on line 95.